bin/ch-run-oci.py.in

```diff
@@ -301,9 +300,9 @@ def state_load():
    st.config = json.load(fs.Path(args.bundle + "/config.json").open_("rt"))
    #debug_lines(json.dumps(st.config, indent=2))
 
-   v_min = packaging.version.Version(OCI_VERSION_MIN)
-   v_actual = packaging.version.Version(st.config["ociVersion"])
-   v_max = packaging.version.Version(OCI_VERSION_MAX)
+   v_min = version_parse_simple(OCI_VERSION_MIN)
+   v_actual = version_parse_simple(st.config["ociVersion"])
+   v_max = version_parse_simple(OCI_VERSION_MAX)
```

Comment:
```suggestion
   v_min = version_parse_oci(OCI_VERSION_MIN)
   v_actual = version_parse_oci(st.config["ociVersion"])
   v_max = version_parse_oci(OCI_VERSION_MAX)
```

The method name can imply general use. The method appears to be intended for OCI version parsing only. Perhaps the method name should be a little more pointed?